cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -16,13 +16,12 @@
 
 package co.cask.cdap.gateway.handlers;
 
-import co.cask.cdap.app.store.Store;
-import co.cask.cdap.app.store.StoreFactory;
 import co.cask.cdap.common.conf.Constants;
-import co.cask.cdap.config.DashboardStore;
-import co.cask.cdap.config.PreferencesStore;
+import co.cask.cdap.common.exception.AlreadyExistsException;
```

Comment:
Usually there is no extra package for grouping Exceptions (e.g. `java.lang.IllegalArgumentException`, not `java.lang.exception.IllegalArgumentException`.